### ISSUE TITLE
Storm Drain Table Widgets

### ISSUE THREAD

**Issue by FLO-2DKaren** [FLO-2DSoftware/qgis-flo-2d-plugin#1314]
https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/20424460/cad06f2a-57d2-49f9-9b80-2dc552539230

**Comment by rpachaly** [comment]
Done on https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/pull/1322:


https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/c6dcf1fd-283a-49cc-b24e-dd1c751e2913

**Comment by FLO-2DKaren** [comment]
Hi Robson,

The date field is not necessary.  It can be NULL

I think we do need the hours:min for the time field because that's what all my projects show.  

In the import export code, we can import inp files with a time series that has no date.  

Do you want to modify the tables so that field is not necessary?

**Comment by FLO-2DKaren** [comment]
![image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/20424460/6cc651df-a8e5-4a53-858f-2fc94abd7011)

**Comment by rpachaly** [comment]
Done! @FLO-2DJJ, you were right about the SU tables. I fixed them. Please, merge it if everything is looking good. 

https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/f7a25891-fec7-4046-b56a-c1fdafdd17b8

**Comment by FLO-2DNoemi** [comment]
We need to figure out the best way to synchronize data and time (INP) with
SIMUL (CON.DAT) and external BC (INP). In previous versions of the plugin,
we grayed date and time out in the INP. The plugin calculated end date and
time internally  as the current date and zero time (START date and time)  +
simul time. This was exported to the SWMM.INP in the fields for start date
and time and end date and time.

Issues with this hardwired synchronization, the BC tables for external flow
also use date and time, and they need to be synchronized to the date and
time from the INP to correctly impose them during runtime. I spent a lot of
time reviewing this for previous plugins, and analyzing the potential
different scenarios, and at that time we decided the best option was to use
SIMUL from CON.DAT file, use current date and zero time (START date and
time in INP), and calculate end date and time using SIMUL, the user needs
to adjust BC to match the date and time. We can use a different approach if
you want but just be aware of the ramifications of this issue.
